src/store/reducers/genres.js

```diff
@@ -0,0 +1,39 @@
+import { GET_GENRES_PENDING, GET_GENRES_SUCCESS, GET_GENRES_ERROR } from '../actionTypes';
+import { normalizeData } from '../../utils';
+
+const defaultState = {
+  byId: {},
+  isLoading: false,
+};
+
+export const genres = (state = defaultState, { type, payload }) => {
+  switch (type) {
+    case GET_GENRES_PENDING:
+      return { ...state, isLoading: true };
+    case GET_GENRES_SUCCESS:
+      return {
+        ...state,
+        byId: normalizeData(payload.genres),
+        isLoading: false,
+      };
+    case GET_GENRES_ERROR:
+      return { ...state, isLoading: false };
+    default:
+      return state;
+  }
+};
+
+export const selectGenres = (store) => store.genres.byId;
+export const selectGenresByIds = (store, ids) => {
+  const result = [];
```

Comment:
you can simplify this selector.
1. get genres as array (using `selectGenres` selector)
2. filter this array using `ids.includes(genreId)`